File: golang/output/formatedText.go
package main

import "fmt"

func main() {
  var name string = "Jhon Doe"
  var age int = 34

  fmt.Println("Age of", name, "is", age)
 //  fmt.Println("Age of " + name + " is " + age) ->  it will work if age is of type string.

  fmt.Printf("Age of %s is %d\n", name, age)
}

File: java/output/FormatedText.java
class FormatedText{
  public static void main(String[] args){
    String name = "Jhon Doe";
    int age = 34;

    System.out.println("Age of " + name + " is " + age);
    /*
        printf() help to easily format the string.
        %s -> String
        %d -> int
        %f -> float
        %c -> char
        %b -> boolean
     */

    System.out.printf("Age of %s is %d%n", name, age);
  }
}
